### ISSUE TITLE
proposal: rule for limit arguments in function call

### ISSUE THREAD

**Issue by luizbills** [jfmengels/eslint-plugin-fp#46]
Something like this:

```js
const create = function (context) {
  return {
    CallExpression (node) {
      const limit = context.options[0]
      if (limit >= 0 && node.arguments.length > limit) {
        context.report({
          node,
          message: 'invalid call expression (too many arguments)',
        })
      }
    }
  }
}
```

```js
// example if configured to accept only 1 argument or less
func () // ok
func (1) // ok
func (1, 2) // error

// example if configured to accept only 3 argument or less
func ([ 1, 2 ], 3) // ok
func ([ 1, 2 ], 3, 4, () => 5) // error
```
